Makefile

```diff
@@ -22,7 +22,7 @@ endif
 
 
 GOTEST_MIN = go test -v -timeout 30s
-GOTEST = $(GOTEST_MIN) -race
+GOTEST = CGO_ENABLED=1 $(GOTEST_MIN) -race
```

Comment:
Why is this added? This will apply to all packages tested. If it is actually needed we need to isolate it to where it actually is needed.

Reply:
this was because I ran `make` and I got an error saying that `CGO_ENABLED` needed to be enabled to run `-race`. I've just enabled it locally instead.

Reply:
I'm still seeing this change. Are you still planning to push a revert here?

Reply:
🤦 , thank you!